### PR TITLE
Adds Vim 8 terminal highlighting

### LINKED ISSUE #125: Add Vim 8 terminal highlighting
Vim 8 comes with a built-in terminal (`:terminal`). The default terminal colors don't look quite right.
screen shot 2018-06-19 at 8 03 19 am

### PR DESCRIPTION
**Pull request by cg433n**
Resolves #125

### CODE REVIEW

**Comment by kooparse** [comment]
I tried it, but it seems that the color don't really match the iTerm one.

With Vim 8:
screen shot 2018-07-03 at 12 04 16 pm

On iTerm:
screen shot 2018-07-03 at 12 04 34 pm

But it is perhaps my config... :)

**Comment by cg433n** [comment]
@kooparse Can you confirm that you were running Vim 8 in iTerm using the `cg433n/nord-vim` fork of nord-vim?

**Comment by kooparse** [comment]
@cg433n Yes I can confirm that :)

vim 8.1 on the left and neovim on the right. 
screen shot 2018-07-05 at 10 33 50 am

**Comment by delphinus** [comment]
In my env, Vim shows valid colors in terminal. Perhaps your vimrc makes a problem?

my reproducible way

* iTerm2 Build 3.2.0beta6
* MacVim 8.1-149

```sh
mkdir /tmp/nord-vim
cd /tmp/nord-vim
git clone https://github.com/cg433n/nord-vim
vim -N -u vimrc -i NONE
```

`/tmp/nord-vim/vimrc`

```vim
set runtimepath+=/tmp/nord-vim/nord-vim
colorscheme nord
```

**Comment by kooparse** [comment]
It is my config, we should merge it then 👍

**Comment by cg433n** [comment]
I've tested with Vim 8 and iTerm and Terminal.app. Everything looks Ok.

**Comment by arcticicestudio** [comment]
@cg433n Thanks for this PR and also everyone else involved for the feedback :+1:

The changes looking good to me, I'll do a quick test on the weekend and merge it to get the next release version out asap.

**Comment by jbadavis** [comment]
Hey @arcticicestudio, could this be merged?

**Comment by arcticicestudio** [comment]
@jbadavis Sorry, this is one of the hundreds issues I've planned to review earlier, but sometime real life keeps you busy. I'll set a reminder and will definitely do it this weekend.

/remind me to review nord-vim#126 until 2019-01-27

**Comment by arcticicestudio** [comment]
@jbadavis Sorry, this is one of the hundreds issues I've planned to review earlier, but sometimes real life keeps you busy. I'll set a reminder and will definitely do it this weekend.

/remind me to review this PR in 3 days

File: colors/nord.vim [hunk 1]
```diff
@@ -158,6 +158,11 @@ call s:hi("healthError", s:nord11_gui, s:nord1_gui, s:nord11_term, s:nord1_term,
 call s:hi("healthSuccess", s:nord14_gui, s:nord1_gui, s:nord14_term, s:nord1_term, "", "")
 call s:hi("healthWarning", s:nord13_gui, s:nord1_gui, s:nord13_term, s:nord1_term, "", "")
 call s:hi("TermCursorNC", "", s:nord1_gui, "", s:nord1_term, "", "")
+"
```
Comment (arcticicestudio) [inline colors/nord.vim]: Please replace this commented line with a blank line.

```suggestion
-"

```

**Comment by sivakov512** [comment]
Hey @arcticicestudio, could this be merged?

**Comment by arcticicestudio** [comment]
@sivakov512 I've reviewed the PR and requested some changes. It'll be merged as soon as the PR author resolved them.

**Comment by arcticicestudio** [comment]
When using vim-plug the latest changes can be used by [defining the `develop` branch as plugin origin](https://github.com/arcticicestudio/nord-vim#via-pluginruntimepath-manager).

**Pull request merged**